backend/prompt_studio/prompt_studio_core/prompt_studio_helper.py

```diff
@@ -785,6 +834,25 @@ def dynamic_indexer(
                 f"Error while indexing '{doc_name}'. {str(e)}"
             ) from e
 
+    @staticmethod
```

Comment:
Can we move all polling related methods to a seperate util? I hope we might use it to some geenric purposes. CommonUtil might be optimal.

Reply:
We're planning to move from polling to pubsub. I'll have this mind while implementing pubsub.